addons/website/static/src/xml/website.editor.xml

```diff
@@ -164,8 +164,7 @@
 
     <!-- Editor Toolbar -->
     <t t-extend="web_editor.toolbar">
-        <t t-jquery="#list .oe-toggle-checklist" t-operation="replace"/>
-        <t t-jquery="#list" t-operation="append">
+        <t t-jquery="#list .oe-toggle-checklist" t-operation="after">
```

Comment:
This used to remove the `checklist` button but with this you reintroduced it. I didn't see that in the spec though.
![image](https://github.com/odoo/odoo/assets/24205914/6c64da28-6dd3-4ff9-bbb9-1619898d2137)
Why did you include this change?

I see there was a [discussion](https://github.com/odoo/odoo/pull/75639#discussion_r705346101) when this was introduced, indicating that getting rid of the checklist button was absolutely on purpose.

Now, if we don't want the animate button outside of website, then IMO the proper way to fix it would be to extend the editor toolbar in a new template and use that as `toolbarTemplate` in the wysiwyg configuration for website.

cc @bvr-odoo @qsm-odoo

Reply:
@shsa-odoo friendly reminder

cc @dmo-odoo

Reply:
@Zynton the issue with extend here `<t t-extend="web_editor.toolbar">` is that it reflects changes everywhere which is the reason I made that change as when the animate button replaces the checklist button from the toolbar in the website because of this [code](https://github.com/odoo/odoo/blob/16.0/addons/website/static/src/xml/website.editor.xml#L167) the checklist is removed completely from the toolbar. It won't recover when I remove the animate button from the toolbar. see in runbot 16.0 [video](https://www.awesomescreenshot.com/video/25485499?key=918688005774c56da9700e1ae0344555) once the animate button is removed by my code the floating toolbar will not have a checklist option see [video](https://www.awesomescreenshot.com/video/25486324?key=4a8910e9ee7b4df0178b3afc3515f1b0), so IMO this should be in spec, that  the floating toolbar should contain checklist button as well when the animate button is removed. I hope this justify the changes I made or we can do something else.
cc @dmo-odoo what is your opinion on this ?

Reply:
Ah I think I see what you're saying.

The checklist was removed on purpose, but you're saying it wasn't on purpose that it was also removed from the floating toolbar, which I can hear as an argument given this is code from `website` so it shouldn't affect other modules. But with your code here you're reintroducing it in `website` as well, which is also not good.

The real issue here is that this template should have been a primary extension. @dmo-odoo is there actually a way to do that in stable?

Reply:
@shsa-odoo @Zynton You are both right. The hiding of the checklist button indeed looks to be a side effect of another PR which was not intended. Fixing it would require to retarget the extension of the template to a new identifier through something like `<t t-name="website.toolbar" t-extend="web_editor.toolbar">`. However, doing so would require thorough testing to be sure we are not missing any piece of code in website that would still be targeting the wrong template or would implicitly rely on the fact that it used to be the same template. Considering the behavior of the checklist button has been like this since the new editor was introduced in 15.0, I think the risks of making a mistake while changing that behavior actually outweighs the potential gain of fixing the issue, especially considering no customer ever reported the issue AFAIK. Because of that, I suggest moving the changes related to that particular issue into a separate PR on master. Thanks for noticing the issue!

Reply:
@dmo-odoo I looked into it, this issue has been fixed from saas-16.4 [here](https://github.com/odoo/odoo/blob/saas-16.4/addons/website/static/src/xml/web_editor.xml#L39) since we have introduced owl. So should we be working on this matter ?

Reply:
@shsa-odoo Oh ok, great, thanks! Then I think we should do nothing. I don't want to change it before master for the reasons I already mentioned above, and you just discovered that it is not needed in master, so we don't need to do anything for that particular issue. You can remove the relevant code from this PR, thanks !

Reply:
@dmo-odoo ok sure so I will revert the changes that reintroduced the checklist button for the website and simply move with removing the animate button from the floating toolbar.

Reply:
@dmo-odoo made changes, runbot is green , we are good to go. Thanks!